contactaccessrights/CRM/Contactaccessrights/Helper/ContactActionsMenu/ContactAccessActionGroup.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+use CRM_HRContactActionsMenu_Component_Group as ActionsGroup;
+use CRM_Contactaccessrights_BAO_Rights as ContactRights;
+use CRM_HRCore_CMSData_UserPermissionInterface as CMSUserPermission;
+use CRM_Contactaccessrights_Service_ACL as ACLService;
+use CRM_HRContactActionsMenu_Component_GroupButtonItem as ActionsGroupButtonItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_GroupTitleToolTipItem as GroupTitleToolTipItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_NoStaffToolTipItem as NoStaffToolTipItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_GenericListItem as GenericListItem;
+use CRM_HRContactActionsMenu_Component_ParagraphItem as ParagraphItem;
+
+class CRM_Contactaccessrights_Helper_ContactActionsMenu_ContactAccessActionGroup {
+
+  /**
+   * @var array
+   */
+  private $contactUserInfo;
+
+  /**
+   * @var ContactRights
+   */
+  private $contactRights;
+
+  /**
+   * @var ACLService
+   */
+  private $aclService;
+
+  /**
+   * @var CMSUserPermission
+   */
+  private $cmsUserPermission;
+
+  /**
+   * CRM_Contactaccessrights_Helper_ContactActionsMenu_ContactAccessActionGroup constructor.
+   *
+   * @param array $contactUserInfo
+   * @param ContactRights $contactRights
+   * @param CMSUserPermission $cmsUserPermission
+   * @param ACLService $aclService
+   */
+  public function __construct(
+    $contactUserInfo,
+    ContactRights $contactRights,
+    CMSUserPermission $cmsUserPermission,
+    ACLService $aclService
+  ) {
+    $this->contactUserInfo = $contactUserInfo;
+    $this->contactRights = $contactRights;
+    $this->cmsUserPermission = $cmsUserPermission;
+    $this->aclService = $aclService;
+  }
+
+  /**
+   * Gets Contact Access Menu Group with menu items already
+   * added.
+   *
+   * @return ActionsGroup
+   */
+  public function get() {
+    $actionsGroup = new ActionsGroup($this->getGroupTitle());
+    $hasAccessCivicrm = $this->hasPermission(['access CiviCRM']);
```

Comment:
`$this->hasPermission(['access CiviCRM']);` is readable is enough. Do we really need to store this in a variable that is only used once?